reactivesocket-cpp/src/ReactiveStreamsCompat.h

```diff
@@ -2,7 +2,7 @@
 
 #pragma once
 
-#include "reactive-streams-cpp/ReactiveStreams.h"
+#include <ReactiveStreams.h>
```

Comment:
The point is that with a single header file, the project name is the file name. So we don't need the extra layer. If forked then they would rename the header file. Since this project name is 'ReactiveStreams'.